addons/web/static/src/js/views/kanban/kanban_model.js

```diff
@@ -405,6 +472,19 @@ var KanbanModel = BasicModel.extend({
         }
         return Promise.resolve();
     },
+    _reloadProgressBarFromRecord: function (recordID, def) {
```

Comment:
missing docstring

Reply:
This method is a ghost :-) I will remove it.